scripts/check-poi.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+
+QUERY_NODE_STATUS_ENDPOINT=$1
+ETHEREUM_NODE=$2
+EPOCH=$3
+INDEXER=$4
+
+if [[ -z "$QUERY_NODE_STATUS_ENDPOINT" ]] || [[ -z "$EPOCH" ]] || [[ -z "$INDEXER" ]] || [[ -z "$ETHEREUM_NODE" ]]; then
+  echo "Usage: $0 <query-node-status-endpoint> <ethereum-node> <epoch-number> <indexer-address>"
+  echo
+  echo "Example: $0 http://localhost:8030/graphql http://localhost:8545/ 29 0x0000000000000000000000000000000000000000"
+  exit 1
+fi
+
+if ! (which http &>/dev/null); then
+  echo "The 'http' command (https://httpie.io/) must be installed for this script to work."
+  exit 1
+fi
+
+CURRENT_EPOCH=$(http -b post https://gateway.network.thegraph.com/network query='{ graphNetworks { currentEpoch } }' | jq '.data.graphNetworks[0].currentEpoch')
```

Comment:
Maybe we should fetch this from the contracts in case the current epoch is not stored yet on the subgraph